llvm/test/CodeGen/X86/select-phi-s16-fp.ll

```diff
@@ -0,0 +1,110 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -global-isel=0 -mcpu=generic -mtriple=x86_64-apple-darwin %s -o - | FileCheck %s
```

Comment:
`-global-isel=0` is redundant.

Reply:
I wanted to make it explicit in case we make progress on the GISel port.
GISel does not have this issue and if we run this example through this framework, it doesn't test what it is meant to.